typescript/sdk/src/token/deploy.hardhat-test.ts

```diff
@@ -68,38 +94,61 @@ describe('TokenDeployer', async () => {
   });
 
   describe('ERC20WarpRouterReader', async () => {
```

Comment:
how annoying would it be to refactor the tests into a file per class

Reply:
Not annoying at all. it was on my todo list :)